vector/build.gradle

```diff
@@ -177,11 +177,11 @@ android {
         // Tchap: Hide voice message recorder button
         buildConfigField "Boolean", "SHOW_VOICE_RECORDER", "false"
 
-        // Tchap: Disable add messages reaction
-        buildConfigField "Boolean", "SHOW_ADD_MESSAGE_REACTION", "false"
+        // Tchap: Enable add messages reaction
```

Comment:
Just a note for other things like this: we should have had a more generic message so we didn't have to update it by changing the flag value